src/components/message.js

```diff
@@ -56,7 +62,8 @@ export default class FormattedMessage extends Component {
   }
 
   render() {
-    const {formatMessage, textComponent: Text} = this.context.intl;
+    const {formatMessage = defaultFormatMessage, textComponent: Text = 'span'} =
```

Comment:
This is going to break with React-Native.
See https://github.com/yahoo/react-intl/wiki/Components#using-react-intl-with-react-native

I wonder if there is simple way to return a <Text /> in RN environment.  @redonkulus do you have any thoughts on that?

Reply:
I'm not entirely comfortable with forking in the library for different environments but I can understand the concern. Unfortunately, I'm not sure if there is an environment variable or flag we can use to conditionally add the default node.

Reply:
Actually I don't think we will be able to default to a <RN.Text /> component in React Native without adding an explicit dependency on the "react-native" package. The best thing we can do here is to show a helpful error message in dev.

Reply:
@giautm We could return a string, but since this library works with older versions of React, I wouldn't want to break existing applications. If we decide to bump major for the new React Context API, then we could look into making this change here too. Please file an issue outlining the enhancement changes and we can discuss with the community there. Thanks!